godot-core/src/macros.rs

```diff
@@ -315,52 +490,53 @@ macro_rules! gdext_virtual_method_callback_inner {
 /// Returns a C function which acts as the callback when a virtual method of this instance is invoked.
 //
 // Note: code duplicated with gdext_virtual_method_callback
+// there are no static virtual methods from what i can tell.
```

Comment:
This could be stated with a bit more confidence, because what would that even mean? `virtual` means dynamic polymorphism; the function to be called is selected at runtime based on the runtime type of the `self`/`this` pointer. But `static` by definition does not have one.